src/xmipp/applications/scripts/deep_denoising/batch_deep_denoising.py

```diff
@@ -105,9 +107,13 @@ def run(self):
     else:
       raise Exception("Error, --mode must be training or denoising")
 
+  def getMdSize(self, filename):
```

Comment:
This looks like sth that should be in our 'core'.

Reply:
I agree. I think it was a fixup I did because it was not in xmipp but in scipion and apparently, we are no longer allowed to use scipion within xmipp

Reply:
I agree, @rsanchezgarc what do you think? Should we generalize it?

Reply:
I guess it is a good idea

Reply:
I refactorized this (among oher metadata utils) in order to avoid duplicated code. It was also in the plugin. Now it is centralized in (bindings)/xmipp_base.py